models/gpg_key.go

```diff
@@ -72,7 +72,7 @@ func ListGPGKeys(uid int64) ([]*GPGKey, error) {
 // GetGPGKeyByID returns public key by given ID.
 func GetGPGKeyByID(keyID int64) (*GPGKey, error) {
 	key := new(GPGKey)
-	has, err := x.Get(key)
+	has, err := x.ID(keyID).Get(key)
```

Comment:
This is a bug, could we create a new PR to fix this and back port it to v1.9?

Reply:
I actually introduced it in this pr (by accident) - it's not actually in the codebase.